### PR TITLE
Ch1 console games

### PR DESCRIPTION
**Pull request by PavithraY22**
## Location

Specify chapters and sections here.

- Chapter(s): 1
- Section(s): Console games

## Changes

- Created 3 new games for ch1

## Notes

- Add any notes here for the Curriculum Development team

**Add all relevant labels to this PR (e.g. if it's about a practice problem, add the practice problem label)**

### CODE REVIEW

File: guessthepassword.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Directions: The goal of this exercise is to create
+# a game where the user has to guess a certain password that
+# you set and see how many tries it takes for that user to guess correctly
+
+# start with assigning the password to some variable
+pas = str("password")
```
Comment (chrehall68) [inline guessthepassword.py]: you don't actually need the str() since "password" is already a string

File: TowerOfCourage.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Story + Directions: This Summer, an amusement
```
Comment (chrehall68) [inline TowerOfCourage.py]: Not exactly a game.

**Review by chrehall68 — CHANGES_REQUESTED** [review]
It mostly looks good. Keep in mind that, since students will have finished our intermediate course that already deals with functions and classes, you can make the problems a bit harder. I recommend that you try to come up with another example to replace TowerofCourage.py

**Comment by PavithraY22** [comment]
> It mostly looks good. Keep in mind that, since students will have finished our intermediate course that already deals with functions and classes, you can make the problems a bit harder. I recommend that you try to come up with another example to replace TowerofCourage.py

I am not too familiar with classes however I can incorporate them in game if you let me know how to create a certain game with classes. I created another game with functions called "Hangman" so please look into that.

**Review by chrehall68 — APPROVED** [review]
LGTM now.

**Review by BenVN123 — APPROVED** [review]
Looks good.

**Pull request merged**